samples/net/sockets/dumb_http_server: mimxrt1015_evk: build failure

**Describe the bug**
samples/net/sockets/dumb_http_server: mimxrt1015_evk: build failure

Please also mention any information which could help others to understand
the problem you're facing:
 - What target platform are you using?
 - mimxrt1015_evk
 - What have you tried to diagnose or workaround this issue?
 - NA
 - Is this a regression? If yes, have you been able to "git bisect" it to a
   specific commit?
 
**To Reproduce**
Steps to reproduce the behavior:

```
west build -b mimxrt1015_evk samples/net/sockets/dumb_http_server -- -DOVERLAY_CONFIG="overlay-netusb.conf;overlay-zeroconf.conf"
```

**Expected behavior**
build PASS

**Impact**
net functions

**Logs and console output**

```
/home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/soc/nxp/imxrt/imxrt10xx/soc.c: In function 'clock_init':
/home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/soc/nxp/imxrt/imxrt10xx/soc.c:216:39: error: 'kIOMUXC_GPR_ENET1TxClkOutputDir' undeclared (first use in this function); did you mean 'kIOMUXC_GPR_SAI1MClkOutputDir'?
  216 |         IOMUXC_EnableMode(IOMUXC_GPR, kIOMUXC_GPR_ENET1TxClkOutputDir, true);
      |                                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                       kIOMUXC_GPR_SAI1MClkOutputDir
/home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/soc/nxp/imxrt/imxrt10xx/soc.c:216:39: note: each undeclared identifier is reported only once for each function it appears in
ninja: build stopped: subcommand failed.

INFO    - /home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/twister-out/mimxrt1015_evk/samples/net/sockets/dumb_http_server/sample.net.sockets.dumb_http_server.netusb/build.log
INFO    - 782 test scenarios (782 test instances) selected, 675 configurations skipped (616 by static filter, 59 at runtime).
INFO    - 105 of 782 test configurations passed (98.13%), 0 failed, 2 errored, 675 skipped with 0 warnings in 209.65 seconds
INFO    - 0 test configurations executed on platforms, 107 test configurations were only built.
INFO    - Saving reports...
INFO    - Writing JSON report /home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/twister-out/twister.json
INFO    - Writing xunit report /home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/twister-out/twister.xml...
INFO    - Writing xunit report /home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/twister-out/twister_report.xml...
INFO    - -+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
INFO    - The following issues were found (showing the top 10 items):
INFO    - 1) samples/net/sockets/dumb_http_server/sample.net.sockets.dumb_http_server.netusb on mimxrt1015_evk error (Build failure)
INFO    - 2) samples/net/sockets/dumb_http_server/sample.net.sockets.dumb_http_server.netusb_zeroconf on mimxrt1015_evk error (Build failure

```

**Environment (please complete the following information):**
 - OS: (e.g. Linux, MacOS, Windows)
 - Toolchain (e.g Zephyr SDK, ...)
 - Commit SHA or Version used:  v3.6.0-2754-g0eb4dc53525d

--- a/dts/arm/nxp/nxp_rt1010.dtsi
+++ b/dts/arm/nxp/nxp_rt1010.dtsi
@@ -115,7 +115,7 @@
 			clocks = <&ccm IMX_CCM_FLEXSPI_CLK 0x0 0x0>;
 		};
 
-		/* Remove SEMC, it does'nt exist on RT1010 */
+		/* Remove SEMC, it doesn't exist on RT1010 */
 		/delete-node/ semc0@402f0000;
 
 		/* Fixup LPI2C1 and LPI2C2, they have different base addr on RT1010 */
@@ -238,7 +238,7 @@
 		/delete-node/ flexpwm@403e4000;
 		/delete-node/ flexpwm@403e8000;
 
-		/* Remove Ethernet, it does'nt exist on RT1010 */
+		/* Remove Ethernet, it doesn't exist on RT1010 */
 		/delete-node/ enet@402d8000;
 
 		/* Fixup USB it has different base addr and interrupt numbers on RT1010 */
@@ -253,14 +253,14 @@
 			status = "disabled";
 		};
 
-		/* Remove USB2, it does'nt exist on RT1010 */
+		/* Remove USB2, it doesn't exist on RT1010 */
 		/delete-node/ usbd@402e0200;
 
 		/* Remove USDHC, they don't exist on RT1010 */
 		/delete-node/ usdhc@402c0000;
 		/delete-node/ usdhc@402c4000;
 
-		/* Remove CSI, it does'nt exist on RT1010 */
+		/* Remove CSI, it doesn't exist on RT1010 */
 		/delete-node/ csi@402bc000;
 
 		/* Remove FLEXCAN, they don't exist on RT1010 */
@@ -268,7 +268,7 @@
 		/delete-node/ can@401d4000;
 		/delete-node/ can@401d8000;
 
-		/* Remove WDOG2, it does'nt exist on RT1010 */
+		/* Remove WDOG2, it doesn't exist on RT1010 */
 		/delete-node/ wdog@400d0000;
 
 		/* Fix SAI1, 3, it has different base addr on RT1010 */
@@ -341,7 +341,7 @@
 			status = "disabled";
 		};
 
-		/* Remove SAI2, it does'nt exist on RT1010 */
+		/* Remove SAI2, it doesn't exist on RT1010 */
 		/delete-node/ sai@40388000;
 	};
 };

--- a/dts/arm/nxp/nxp_rt1015.dtsi
+++ b/dts/arm/nxp/nxp_rt1015.dtsi
@@ -61,6 +61,8 @@
 		/delete-node/ gpio@4200c000;
 		/* RT1015 has only one flexSPI controller */
 		/delete-node/ spi@402a4000;
+		/* Remove Ethernet, it doesn't exist on RT1015 */
+		/delete-node/ enet@402d8000;
 	};
 };
 

